opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/TracerShim.java

```diff
@@ -37,7 +37,7 @@
    * @since 0.1.0
    */
   public TracerShim() {
-    this(Trace.getTracer());
+    this(NoopTrace.newNoopTracer());
```

Comment:
Actually I'd like this one to call `OpenTelemetry.getTracer()` ;)

Reply:
+1 I will update it